src/measure_theory/decomposition/lebesgue.lean

```diff
@@ -58,7 +58,7 @@ Lebesgue decomposition theorem
 noncomputable theory
 open_locale classical measure_theory nnreal ennreal
 
-variables {α β : Type*} [measurable_space α]
+variables {α β : Type*} {m : measurable_space α} {μ ν : measure_theory.measure α}
```

Comment:
Would it make sense to `include m` here, so it doesn't need to appear explicitly throughout? I realise there are a handful of lemmas that don't use it, and `omit`ing it or using sections could be annoying.

Reply:
You're right, `include` is a better way of doing it. The only lemmas for which I needed an `omit` should not be in this file anyway (lemmas about `supr`). I'll deal with them in another PR.